.github/CODEOWNERS

```
@@ -42,23 +42,39 @@
 # Important Integrations
 /Packs/ServiceNow/Integrations/ServiceNowv2/* @Shellyber
 /Packs/QRadar/Integrations/QRadar_v3/* @ilaner
-/Packs/Slack/Integrations/* @amshamah419 @rshunim 
+/Packs/Slack/Integrations/* @amshamah419 @rshunim
 /Packs/SplunkPy/Integrations/SplunkPy/* @ilappe
-/Packs/MicrosoftExchangeOnPremise/Integrations @amshamah419 
+/Packs/MicrosoftExchangeOnPremise/Integrations @amshamah419
```

```suggestion
/Packs/MicrosoftExchangeOnPremise/Integrations/** @amshamah419
```
same change to all folders, otherwise the ownership won't apply to files (we recently missed a PR because of such a mistake)

---

I didn't change this line, its one of your linters... 
Most of the lines don't have asterisks